### PR TITLE
LEP-172 remarks for partner variations to match those for applicant

### PR DESCRIPTION
**Pull request by starswan**
https://dsdmoj.atlassian.net/browse/LEP-172

Convert remarks orchestrator to create remarks for partner data as well as applicant data

### CODE REVIEW

File: app/services/workflows/main_workflow.rb [hunk 1]
```diff
@@ -10,7 +10,23 @@ def call(assessment:, applicant:, partner:)
                              else
                                NonPassportedWorkflow.call(assessment:, applicant:, partner:)
                              end
-        RemarkGenerators::Orchestrator.call(assessment, calculation_output.capital_subtotals.combined_assessed_capital)
+        # we can take the lower threshold from the first eligibility records as they are all the same
+        lower_capital_threshold = assessment.applicant_capital_summary.eligibilities.first.lower_threshold
```
Comment (ripan) [inline app/services/workflows/main_workflow.rb:14]: @starswan Is lower_capital_threshold came for applicant and partner?

File: spec/requests/v6/assessments_controller_spec.rb [hunk 2]
```diff
@@ -1100,7 +1100,7 @@ def cash_transactions(amount)
 
             it "has outgoings_housing_cost" do
               expect(remarks.fetch(:outgoings_housing_cost)).to eq(
-                { unknown_frequency: ["hc-r-1"] },
+                { unknown_frequency: ["hc-r-1", "hc-r-1"] },
```
Comment (ripan) [inline spec/requests/v6/assessments_controller_spec.rb:1103]: @starswan We have duplicate elements in an array. Is it a type?

Comment (starswan) [reply spec/requests/v6/assessments_controller_spec.rb:1103]: No - its some proof that the changes worked and we are getting remarks for partner

File: app/services/workflows/main_workflow.rb (hunk 1 above)
Comment (starswan) [reply app/services/workflows/main_workflow.rb:14]: partner doesn't have 'eligibilities' - only applicant. (yes this is one confusing thing still present in the codebase)

**Review by ripan — APPROVED** [review]

**Pull request merged**